src/components/Modal/EditModal.tsx

```diff
@@ -0,0 +1,68 @@
+import styles from "./modal.module.css";
+import ModalCloseButton from "../common/Modal/ModalButtons/ModalCloseButton";
+import ModalResetButton from "../common/Modal/ModalButtons/ModalResetButton";
+import ModalSubmitButton from "../common/Modal/ModalButtons/ModalSubmitButton";
+import ModalForm from "../common/Modal/ModalForm/ModalForm";
+import { useState } from "react";
+import { EditModalType } from "../../types/modal/editModal.type";
+import { MockMoviesArray } from "../../mockData/movies";
+import { MovieType } from "../../types/movie/movie.type";
+
+function EditModal({ toggle, handleCloseButton, movieId }: EditModalType) {
```

Comment:
This component looks just like `AddModal` component, please make one common

actually you should have one form for these two components